### ISSUE TITLE
10bit force transcode should differentiate between h264 and h265

### ISSUE THREAD

**Issue by Draic** [croneter/PlexKodiConnect#367]
My Client does hardware decoding of 10bit in h265 but not in h264. At the moment, if I tell him to force transcode 10bit content, it will do so regardless of the codec of the files. I would like to split the option of force transcode on 10bit content. I my opinion it would be enough, if enabling this option will only force transcode on any files that are h264. As far as I know, any CPU that can hardware decode h265, can also do so if it is 10bit, so there is never a reason why someone would want to force transcode on 10bit here unless he needs to transcode h265 anyways, which has its own option already.

tl;dr
force transcode on 10bit should only apply to files that are h264, not on h265.

**Comment by Draic** [comment]
this is the opposite from the requested feature. Your commit makes it so that the hi10p option only works on h265 but never on h264. It is supposed to be the other way around. Let the force hi10p option only force transcoding on h264 videos

**Comment by croneter** [comment]
Fixed with merging your PR https://github.com/croneter/PlexKodiConnect/commit/3b14373bd30a6b46feb77edde2c4a71da423699a
